env0/resource_environment.go

```diff
@@ -23,27 +23,142 @@ func resourceEnvironment() *schema.Resource {
 			},
 			"name": {
 				Type:        schema.TypeString,
-				Description: "The environment's name",
+				Description: "the environment's name",
 				Required:    true,
 			},
 			"project_id": {
 				Type:        schema.TypeString,
 				Description: "project id of the environment",
 				Required:    true,
+				ForceNew:    true,
 			},
 			"template_id": {
 				Type:        schema.TypeString,
 				Description: "the template id the environment is to be created from",
 				Required:    true,
 			},
+			"workspace": {
+				Type:        schema.TypeString,
+				Description: "the terraform workspace of the environment",
+				Optional:    true,
+				ForceNew:    true,
+				Computed:    true,
+			},
+			"revision": {
+				Type:        schema.TypeString,
+				Description: "the revision the environment is to be run against",
+				Optional:    true,
+				Computed:    true,
+			},
+			"run_plan_on_pull_requests": {
+				Type:        schema.TypeBool,
+				Description: "should run terraform plan on pull requests creations",
+				Optional:    true,
+				Default:     false,
+			},
+			"approve_plan_automatically": {
+				Type:        schema.TypeBool,
+				Description: "should deployments require manual approvals ( defaults to true )",
+				Optional:    true,
+				Default:     true,
```

Comment:
why those are the defaults?
i'd rather us to make another request to fetch the policies and choose from them

Reply:
I just went with whats on the [ticket](https://trello.com/c/vCEIi9gh/3988-terraform-provider-environment-resource-part-1?filter=*)

Reply:
7dd986c

Reply:
e485770